cfripper/rules/ec2_security_group.py

```diff
@@ -37,7 +38,7 @@ def analyse_ingress(
         resource_type: str,
     ):
         if self.non_compliant_ip_range(ingress=ingress):
-            open_ports = list(range(ingress.FromPort, ingress.ToPort + 1))
+            open_ports = list(range(ingress.FromPort or 0, (ingress.ToPort or MAX_PORT_NUMBER) + 1))
```

Comment:
What do you think of this?
```suggestion
            open_ports = list(range(ingress.FromPort or MIN_PORT_NUMBER, (ingress.ToPort or MAX_PORT_NUMBER) + 1))
```